apps/src/lab2/redux/predictLevelRedux.ts

```diff
@@ -49,6 +49,31 @@ export const resetPredictProgress = createAsyncThunk<
   }
 });
 
+export const submitPredictResponse = createAsyncThunk<
```

Comment:
nit: does this need to be an `async` thunk? Do we need to `await` the dispatch?

Reply:
Oh, I guess we don't really. My thinking was that sending a report involves a network request or some other asynchronous stuff, but we don't really need to perform any actions based on the result, so it's probably find to not `await` it. Does that sound right to you?

Reply:
Yep that sounds right, since `dispatch` itself doesn't return anything (or doesn't return a promise AFAIK). Even if we did want to wait for the result to finish processing, we would instead listen for the relevant state update (e.g. some progress value that got updated after the network call), or do something with `extraReducers`.

Reply:
Updated!